packages/devtools_app/lib/src/code_size/code_size_screen.dart

```diff
@@ -94,12 +96,18 @@ class CodeSizeBodyState extends State<CodeSizeBody>
       return Column(
         crossAxisAlignment: CrossAxisAlignment.start,
         children: [
-          TabBar(
-            labelColor: Theme.of(context).textTheme.bodyText1.color,
-            isScrollable: true,
-            controller: _tabController,
-            tabs: tabs,
-            onTap: onTabSelected,
+          Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              TabBar(
+                labelColor: Theme.of(context).textTheme.bodyText1.color,
+                isScrollable: true,
+                controller: _tabController,
+                tabs: tabs,
+                onTap: onTabSelected,
+              ),
+              if (_tabController.index == 1) _buildDiffTreeTypeDropdown(),
```

Comment:
put `1` into a const `diffTabIndex`. It would be nice if we can check on something more concrete than index, like the tab itself for example.